packages/calcite-components/src/components/action-bar/action-bar.tsx

```diff
@@ -212,6 +221,7 @@ export class ActionBar extends LitElement {
     this.mutationObserver?.observe(this.el, { childList: true, subtree: true });
     this.overflowActionsDisabledHandler(this.overflowActionsDisabled);
     this.cancelable.add(this.resize);
+    this.updateSlottedActions();
```

Comment:
Calling this on connect might be unnecessary. I _think_ the updates from mutation observer and changes to `selectionAppearance` should suffice. LMK if I'm off track here.